src/ResourceManagement/Logic/Logic.Tests/InMemoryTests/IntegrationAccountSessionInMemoryTests.cs

```diff
@@ -210,10 +205,6 @@ public void IntegrationAccountSession_Delete_Exception()
                 resourceGroupName: ResourceGroupName,
                 integrationAccountName: "IntegrationAccountName",
                 sessionName: null));
-            Assert.Throws<CloudException>(() => client.Sessions.Delete(
```

Comment:
This assert was removed because the mocking of the server response was incorrect. Delete on an entity that is not present when the request is properly formulated does not result in a 404/Not Found error but in a 204/No Content response. 204/No Content is not an exception and does not belong to this test Fact (IntegrationAccountSession). 204/No Content is already tested in existing test Fact IntegrationAccountSession_Delete_NoContent - the deleted test was further contradicting that.